Extension/src/LanguageServer/client.ts

```diff
@@ -962,35 +975,35 @@ export class DefaultClient implements Client {
         this.updateClientConfigurations();
     }
 
-    async promptSelectCompiler(command: boolean): Promise<void> {
+    async promptSelectCompiler(isCommand: boolean): Promise<void> {
         const selectCompiler: string = localize("selectCompiler.string", "Select Compiler");
         const confirmCompiler: string = localize("confirmCompiler.string", "Yes");
         const settings: OtherSettings = new OtherSettings();
         if (compilerDefaults.compilerPath !== "") {
-            if (!command) {
+            if (!isCommand && (compilerDefaults.compilerPath !== undefined)) {
                 const value: string | undefined = await vscode.window.showInformationMessage(localize("selectCompiler.message", "The compiler {0} was found on this computer. Do you want to configure it for IntelliSense?", compilerDefaults.compilerPath), confirmCompiler, selectCompiler);
                 if (value === confirmCompiler) {
                     compilerPaths = await util.addTrustedCompiler(compilerPaths, compilerDefaults.compilerPath);
                     settings.defaultCompiler = compilerDefaults.compilerPath;
                     compilerDefaults = await this.requestCompiler(compilerPaths);
                     this.updateClientConfigurations();
                 } else if (value === selectCompiler) {
-                    this.handleCompilerQuickPick();
+                    this.handleCompilerQuickPick(true);
                 } else {
                     const setCompiler: string = localize("setCompiler.string", "Set Compiler");
-                    const value: string | undefined = await vscode.window.showInformationMessage(localize("setCompiler.message", "You do not have a compiler configured. Unless you set your own configurations, IntelliSense may not be functional."), selectCompiler);
+                    const value: string | undefined = await vscode.window.showInformationMessage(localize("setCompiler.message", "You do not have a compiler configured. Unless you set your own configurations, IntelliSense may not be functional."), setCompiler);
```

Comment:
Why did this change from selectCompiler to setCompiler?

Reply:
Yeah, I'd prefer they'd all say "Select Compiler", unless we think of a reason to say something different.